Programs/_testembed.c

```diff
@@ -73,14 +73,20 @@ static void init_from_config_clear(PyConfig *config)
 }
 
 
-static void _testembed_Py_Initialize(void)
+static void _testembed_Py_InitializeFromConfig(void)
 {
     PyConfig config;
     _PyConfig_InitCompatConfig(&config);
     config_set_program_name(&config);
     init_from_config_clear(&config);
 }
 
+static void _testembed_Py_Initialize(void)
+{
+   Py_SetProgramName(PROGRAM_NAME);
+   Py_Initialize();
```

Comment:
This function is deprecated. You need the compiler pragma black magic to turn off compiler warnings.

```
_Py_COMP_DIAG_PUSH
_Py_COMP_DIAG_IGNORE_DEPR_DECLS
...
_Py_COMP_DIAG_POP
```

Reply:
`_testembed.c` already has deprecation warnings disabled near the top of the file: https://github.com/python/cpython/blob/6cd0446ef72c6676b292d7f54b1ddb8ae5e1fb8d/Programs/_testembed.c#L25